frontend/src/services/postService.ts

```diff
@@ -2,4 +2,4 @@ import { dataType } from '../components/interfaces/Interface';
 import { getRequest, postRequest } from './apiService';
 
 export const getUserPosts = (id?: string) => getRequest(`${dataType.user}/${id}/${dataType.posts}`);
-export const sendPostToUser = (id: string, body: {}) => postRequest(`${dataType.user}/${id}/${dataType.posts}`, body);
+export const sendPostToUser = (id: string, body: {}) => postRequest(`${dataType.user}/${id}/${dataType.posts}`, body).then(res => res);
```

Comment:
What the point of doing this?

Reply:
I removed it. This part(.then(res => res))